packages/cli/src/constructs/alert-escalation-policy.ts

```diff
@@ -1,33 +1,113 @@
+/** Types of alert escalation strategies */
 // eslint-disable-next-line no-restricted-syntax
 enum AlertEscalationType {
   RUN = 'RUN_BASED',
   TIME = 'TIME_BASED'
 }
 
+/**
+ * Configuration for alert reminders.
+ * Defines how often to send reminder notifications after initial alert.
+ */
 export type Reminders = {
+  /** 
+   * Number of reminder notifications to send (0 to disable).
+   * @defaultValue 0
+   * @minimum 0
+   * @maximum 5
```

Comment:
It looks like `100000` is also a supported value, probably to indicate practically unlimited reminders.

This could therefore perhaps be changed to:
```suggestion
   * @enum [0, 1, 2, 3, 4, 5, 100000]
```

Although ideally we'd put the "unlimited" value behind a constant. Do we want to expose this unlimited value in the docs?